src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -279,6 +279,7 @@ private CookieOptions BuildCookieOptions()
             var context = new CookieSigningOutContext(
                 Context,
                 Options,
+                new AuthenticationProperties(signOutContext.Properties),
```

Comment:
Any reason to flow the properties using a constructor parameter here and a property everywhere else?

Personally, I'd use a parameter everywhere (and probably make the `Properties` property get-only).